<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComWidget</class>
 <widget class="QWidget" name="ComWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblComNo">
       <property name="text">
        <string>串口号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbComNo">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hsComNo">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblBaudBits">
       <property name="text">
        <string>波特率</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBaudBits">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hsBaudBits">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lblParityBits">
       <property name="text">
        <string>校验位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbParityBits">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hsParity">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lblDataBits">
       <property name="text">
        <string>数据位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbDataBits">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hsDataBits">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="lblStopBits">
       <property name="text">
        <string>停止位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbStopBits">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hsStopBits">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnOpenClose">
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>19</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
